### ISSUE TITLE
Strategy.js throws uncaught exception "TypeError: this.q is not a function"

### ISSUE THREAD

**Issue by pidtuner** [GoogleChrome/workbox#2686]
**Library Affected**:
*workbox-cdn*

**Browser & Platform**:
*all browsers*

**Issue or Feature Request Description**:

Using version `6.0.0` in my project <https://pidtuner.com>. You can check up the bug live right now at the site. Source code in:

<https://github.com/pidtuner/pidtuner.github.io>

When developing on a local server works fine, but once I upload to production, this bug appears:

![image](https://user-images.githubusercontent.com/43546068/100863652-29251680-3495-11eb-82da-d9ebd55bd2a6.png)

I also notice that every time I refresh, a new service worker appears to be registered:

![image](https://user-images.githubusercontent.com/43546068/100863733-4954d580-3495-11eb-8891-ee8f0ed80f48.png)

**Comment by jeffposnick** [comment]
Thanks for reporting this.

After debugging the https://pidtuner.com site, I think the issue is due to https://github.com/GoogleChrome/workbox/issues/1674, which... was not resolved prior to the launch of v6. CC: @philipwalton 

(I don't see a `q` property on the `PrecacheStrategy` instance or the `Strategy` prototype, which is what `workbox-precaching` thinks `_handle()` was minified as.)

<img width="508" alt="Screen Shot 2020-12-02 at 1 19 58 PM" src="https://user-images.githubusercontent.com/1749548/100914665-5b3c7580-34a1-11eb-80a4-022c7e200138.png">

**Comment by jeffposnick** [comment]
A short-term fix while remaining on v6.0.0 would be to [switch to the development builds](https://developers.google.com/web/tools/workbox/guides/configure-workbox#configure_debug_builds_vs_production_builds) of Workbox, which doesn't minify the source code (and therefore shouldn't be confused by cross-package symbol minification).

```
// Place this immediately after your importScripts(...) statement.
workbox.setConfig({ debug: true });
```

The development builds are the default when serving from `localhost`, which is why you only see this after deployment.

**Comment by juangburgos** [comment]
I confirm that the `workbox.setConfig({ debug: true });` works, now added to <https://pidtuner.com>. Will update to production when fixed.

Thanks!

**Comment by jeffposnick** [comment]
This should be fixed in the `6.0.2` release.
